guides/common/modules/con_managing-red-hat-subscriptions.adoc

```diff
@@ -19,6 +19,55 @@ The advantage of this is that each organization maintains separate subscriptions
+
+The subscriptions service, hosted at console.redhat.com provides a means for customers to view their subscription usage and can show usage data from both connected and disconnected environments if configured to do so.
+
+.Configuring your system to use the subscription services
+
+Connected (Non-{Project}) users::
+
+As a connected, non-{Project} user, no further action needs to be taken to use the subscriptions service. 
+For more information, see https://console.redhat.com/subscriptions/usage/rhel to track your usage. 
+Registering a system through the guidance in https://docs.redhat.com/en/documentation/subscription_central/1-latest/html-single/getting_started_with_rhel_system_registration/index[Getting Started with RHEL System Registration] is sufficient to ensure the system is properly counted. 
+If necessary, set https://access.redhat.com/articles/system-purpose[System Purpose] attributes to denote SLA and USAGE to better power filtering.
+
+Connected {Project} Users::
+
+Connected {Project} Users should enable the `foreman_rh_cloud` plugin if it hasn’t been enabled already (https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_the_subscriptions_service/proc-installing-satellite-inventory-upload-plugin_assembly-setting-up-subscriptionwatch-ctxt). The foreman_rh_cloud plugin shares inventory with the Hybrid Cloud Console enabling the usage of the subscriptions service to track usage information.
+
+The plugin can be configured to omit data that is not pertinent for subscription purposes such as hostnames & IP addresses.
+
+Disconnected users::
+
+Users who are unwilling or unable to use a directly connected experience can export a summary of their product usage locally and import it into tools like their CMDB, spreadsheet, or other systems to track usage without connecting to an external service.
+
+Disconnected (No {Project})::
+
+As a disconnected user, you can use Red Hat Discovery to query standalone RHEL systems or tools such as {ProjectName} to build a report locally. 
+Alternatively, you can leverage any tool, whether provided by Red Hat or another source, to track your subscription usage.
+
+Some users prefer to review usage reports before sending to the Hybrid Cloud Console. 
+Discovery allows users to upload reports in an ad-hoc fashion.
+
+Disconnected ({Project})::
+
+{Project} users can export usage data in one of two ways:
+
+** Through the `foreman_rh_cloud` plugin, which generates its report locally and allows you the ability to download it. 
+This report is in JSON format and is most suitable for automated scripts or machine ingestion. 
+This report can also be generated through the `foreman-rake` command.
+
+** Through the *Host - Installed Products* report by navigating to *Monitor* > *Reports* > *Report Templates*. 
+This report is in CSV format and is most suitable for importing into a spreadsheet tool of your choosing.
```

Comment:
Report templates can be exported in the format of your choosing, including HTML, YAML, or CSV. It's not only CSV.